settings.gradle

```diff
@@ -25,6 +25,7 @@ include ':backend.native'
 include ':runtime'
 include ':common'
 include ':backend.native:tests'
+include ':backend.native:compiletest'
```

Comment:
Not sure that the whole activity should be named as 'compiletest'. It's rather debugging or debug-ability

Reply:
In theory, we can use JUnit test runner to drive some other tests, and not only debugger-related ones, so I picked a more general name. Should I rename this to `debuggertests`?

Reply:
Made the module name more specific.